examples/scripts/config.js

```diff
@@ -22,4 +22,10 @@ angular.module('examples')
 		});
 
 	}
-]);
\ No newline at end of file
+])
+.config(['$compileProvider',
```

Comment:
Added to be inline with most prod consumers of something like this so that any future use of .scope or .isolatedScope off the angular elements is undefined.